src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComMethodContext.cs

```diff
@@ -50,17 +50,21 @@ public MethodDeclarationSyntax GenerateShadow()
                 // {
                 //    return ((<baseInterfaceType>)this).<MethodName>(<Arguments>);
                 // }
-                return MethodInfo.Syntax.WithBody(
-                    Block(
-                        ReturnStatement(
+                // TODO: Copy full name of parameter types and attributes / attribute arguments for parameters
+                return MethodInfo.Syntax
+                    .WithModifiers(TokenList(Token(SyntaxKind.NewKeyword)))
+                    .WithExpressionBody(
```

Comment:
Arrow expression body makes it easier to handle void methods - no need to worry about the return keyword being there or not.